docs/access-control/globals.mdx

```diff
@@ -6,7 +6,7 @@ desc: Global-level Access Control is specified within each Global's `access` pro
 keywords: globals, access control, permissions, documentation, Content Management System, cms, headless, javascript, node, react, nextjs
 ---
 
-Global Access Control is [Access Control](../overview) used to restrict access to [Global](../globals/overview) Documents, as well as what they can and cannot see within the [Admin Panel](../admin/overview) as it relates to that Global.
+Global Access Control is [Access Control](../overview) used to restrict access to [Global](../getting-started/concepts#globals) Documents, as well as what they can and cannot see within the [Admin Panel](../admin/overview) as it relates to that Global.
```

Comment:
This link should point to the globals configuration page, not the concepts overview:

```suggestion
Global Access Control is [Access Control](../overview) used to restrict access to [Global](../configuration/globals) Documents, as well as what they can and cannot see within the [Admin Panel](../admin/overview) as it relates to that Global.
```

Same as this line: https://github.com/payloadcms/payload/pull/10010/files#diff-b0cb3b6319af3e2dae2e9ddbe20fb2ac53ceed0bb29e39ed68399efea88e182aR9